### PR TITLE
Code Generator for Digital Ocean Cloud Provider

### PR DESCRIPTION
**Pull request by rajitha1998**
# Master PR
**Main milestone:**  Adding Digital Ocean Provider to Code Generation Tool by @mbcse

## Contains:

- Digital Ocean Parser: #96
- Transformer and Generator for Digital Ocean Code Generation: #97
- Tests for Digital Ocean Parser: #98
- Tests for Digital Ocean Transformer: #99 
- Tests for Digital Ocean generator: #100 
- Digital Ocean Services to YML file: #101 
- Digital Ocean Plugin: #102 
- Examples for Digital Ocean NodeCloud-Plugin: #106 
- Updated Documentation for Nodecloud and Generator: #107 
- Final Touches: #108

### CODE REVIEW

**Comment by rajitha1998** [comment]
We need two approvals @leopardslab/nodecloud-committers

**Review by rehrumesh — APPROVED** [review]

**Pull request merged**
